crmprtd/bc_hydro/download.py

```diff
@@ -0,0 +1,116 @@
+''' Downloads data from BC hyrdo
+
+BC Hydro posts a rolling window (3 months) observing hourly data
+once a week. Data is in txt files.
+
+It is recommended to run this script once every 3 months to collect
```

Comment:
Well, if they post once a week, it would be better to run once a week to get updates. Then there's a very loose connection between the update cycle and the *you-will-lose-data* cycle. If we recommend to run once a week, then monitoring can be much more casual.